cabal-install-solver/ChangeLog.md

```diff
@@ -0,0 +1 @@
+Please see https://hackage.haskell.org/package/cabal-install/changelog
```

Comment:
I'm curious if the plan is to always have one changelog for the two packages, or if going forward `cabal-install-solver` will get a proper changelog. If it's the latter, I'd probably add a heading `## Pre-3.10` before the current phrase, or something like that.

Reply:
Yeah, less changelogs seems more convenient to me.